includes/class-wc-taxjar-customer-sync.php

```diff
@@ -142,48 +148,110 @@ public function add_customer_meta_fields( $user ) {
 			</table>
-		<?php
+			<?php
 		endforeach;
 	}
 
+	/**
+	 * Saves tax exemption user meta if necessary
+	 *
+	 * @param $user_id - Id of user to update
+	 */
 	public function save_customer_meta_fields( $user_id ) {
-		if ( ! apply_filters( 'taxjar_current_user_can_edit_customer_meta_fields', current_user_can( 'manage_woocommerce' ), $user_id ) ) {
+		$has_permission = current_user_can( 'manage_woocommerce' ) || current_user_can( 'edit_users' );
+		if ( ! apply_filters( 'taxjar_current_user_can_edit_customer_meta_fields', $has_permission, $user_id ) ) {
 			return;
 		}
 
-		$save_fields = $this->get_customer_meta_fields();
-		$change = false;
-		foreach ( $save_fields as $fieldset ) {
-			foreach ( $fieldset['fields'] as $key => $field ) {
-				if ( isset( $field['type'] ) && 'multi-select' === $field['type'] ) {
-				    $prev_value = get_user_meta( $user_id, $key, true );
-				    if ( empty( $_POST[ $key ] ) ) {
-				        $exempt_regions = '';
-                    } else {
-				        $exempt_regions = array_map( 'wc_clean',  $_POST[ $key ] );
-				        $exempt_regions = implode( ',', $exempt_regions );
-                    }
-
-				    if ( $exempt_regions != $prev_value ) {
-					    update_user_meta( $user_id, $key, $exempt_regions );
-					    $change = true;
-                    }
-				} elseif ( isset( $_POST[ $key ] ) ) {
-					$prev_value = get_user_meta( $user_id, $key, true );
-					$new_value = wc_clean( $_POST[ $key ] );
-					if ( $prev_value != $new_value ) {
-						update_user_meta( $user_id, $key, $new_value );
-						$change = true;
-                    }
-				}
-			}
+		update_user_meta( $user_id, 'tax_exemption_type', $this->get_posted_exemption_type() );
+		update_user_meta( $user_id, 'tax_exempt_regions', $this->get_posted_exempt_regions() );
+
+		do_action( 'taxjar_customer_exemption_settings_updated', $user_id );
+	}
+
+	/**
+	 * Checks if exemption type has changed when saving a user
+	 *
+	 * @param $user_id - Id of customer
+	 *
+	 * @return bool - Whether or not the exemption type has been changed
+	 */
+	public function has_exemption_type_changed( $user_id ) {
+		$saved_exemption_type = self::get_saved_exemption_type( $user_id );
+		$new_exemption_type   = $this->get_posted_exemption_type();
+		if ( $new_exemption_type !== $saved_exemption_type ) {
+			return true;
+		}
+
+		return false;
+	}
+
+	/**
+	 * Gets the submitted tax exemption type during user save
+	 *
+	 * @return array|string - value to save
+	 */
+	public function get_posted_exemption_type() {
+		return wc_clean( $_POST['tax_exemption_type'] );
+	}
+
+	/**
+	 * Checks if exempt regions have changed for a user
+	 *
+	 * @param $user_id - Id of user to check
+	 *
+	 * @return bool - Whether or not the exempt regions have changed
+	 */
+	public function have_exempt_regions_changed( $user_id ) {
+		$saved_exempt_regions = self::get_saved_exempt_regions( $user_id );
+		$new_exemption_type   = $this->get_posted_exempt_regions();
```

Comment:
@dallendalton Naming tweak: `$new_exemption_type` -> `$new_exempt_regions`

Reply:
Nice catch, this has been fixed.